### PR TITLE
section 9 design

### PR DESCRIPTION
**Pull request by Plouc314**

### CODE REVIEW

File: template/9-design.md [hunk 1]
```diff
@@ -1,36 +1,68 @@
...
+### Database
 
-*Where is it stored?*
+Google Cloud Firestore, a No-SQL database, is used for data storage. It supports live updates and limited queries. Firestore is employed both for long-term data storage like user, vehicle, and balloon data and real-time updates like current user position or a flight status. Firestore-managed cache caters to the application's caching needs.
 
-*How is it shared/copied/cached?*
+### Push notification
 
-## Security Considerations
+Firebase Cloud Messaging (FCM) is used for push notifications. Notifications are triggered on receiving a message or the initiation of a flight.
```
Comment (mathieumognol) [inline template/9-design.md]: Really?

File: template/9-design.md [hunk 2]
```diff
@@ -1,36 +1,68 @@
...
+
+## Data Model
+
+The Firestore database is structured much like a SQL database with specific collections (tables) defined for each schema and relations established among them. The system also supports versioning of schemas enabling smooth updates and maintenance of old data.
+
+Database schemas comprise:
+- Balloons, vehicles, baskets schemas for inventory.
+- Flight, flight types, flight traces, and flight-member relation schemas for flight lifecycle management.
+- ocations, message group, and message schemas for real-time updates and message storage.
```
Comment (mathieumognol) [inline template/9-design.md]: missing the "L" of Locations

**Review by mathieumognol — APPROVED** [review]
LGTM

File: template/9-design.md (hunk 1 above)
Comment (Plouc314) [reply template/9-design.md]: It is the design of a product, not our current application, so ideally yes, it should have a push notification service.

**Pull request merged**